<?xml version="1.0"?>
<launch>
<arg name="hokuyo_flag" default="true" /> <!-- set to true if i want to record-->

	<node name="urg_node" pkg="urg_node" type="urg_node" output="screen" if = "$(arg hokuyo_flag)">
    <param name="ip_address" value="172.16.0.10"/>
 <!--    <param name="serial_port" value="/dev/ttyACM0"/>-->
<!--     <param name="serial_baud" value="115200"/>-->
    <param name="frame_id" value="laser"/>
    <param name="calibrate_time" value="true"/>
    <param name="publish_intensity" value="false"/>
    <param name="publish_multiecho" value="false"/>
    <param name="angle_min" value="-1.5707963"/>
    <param name="angle_max" value="1.5707963"/>
  </node>

</launch>
